src/routes/favoriteHoteles.js

```diff
@@ -0,0 +1,72 @@
+import { Router } from "express";
+import { param, query } from "express-validator";
+import { validateJwt } from "../middleware/validate-jwt.js";
+import { isAdminLogged } from "../middleware/is-logged.js";
+import { validateRequestParams } from "../middleware/validate-request-params.js";
+
+import {
+  favoritesGet,
+  getFavoriteById,
+  putFavorite,
+  favoriteDelete,
+  favoritePost,
+} from "../controller/favoriteHotel.controller.js";
+
+const router = Router();
+
+router.get(
+  "/",
+  [
+    query("limite")
+      .optional()
+      .isNumeric()
+      .withMessage("El límite debe ser un valor numérico"),
+    query("desde")
+      .optional()
+      .isNumeric()
+      .withMessage("El valor desde debe ser numérico"),
+    validateRequestParams,
+  ],
+  favoritesGet
+);
+
+router.get(
+  "/:id",
+  [param("id").isMongoId(), validateRequestParams],
+  getFavoriteById
+);
+
+router.put(
+  "/:id",
+  [
+    validateJwt,
+    isAdminLogged,
+    param("id").isMongoId(),
+    validateRequestParams,
+  ],
+  putFavorite
+);
```

Comment:
Este endpoint no deberia existir, como ya dije antes, tendremos, create, delete, and get. ademas deberiamos crear get all likes by hotel id.
```suggestion
```